### PR TITLE
Cam 13869

### PR DESCRIPTION
**Pull request by MaxTru**
Closes https://github.com/camunda/camunda-modeler/issues/2357

### CODE REVIEW

**Comment by MaxTru** [comment]
Will be closed via https://github.com/camunda/camunda-modeler/issues/2357 , review assigned to @volkergersabeck

File: content/reference/forms/camunda-forms/_index.md [hunk 1]
```diff
@@ -27,38 +25,121 @@ Camunda Forms allow you to easily design and configure forms and then embed them
...
+
+A number field allowing the user to read and edit numeric data.
+
+{{< img src="img/form-number.svg" title="Form Number Symbol" >}}
+
+A number can be configured using the following configuration properties:
+
+* **Field Label**: Label displayed on top of the number field.
+* **Field Description**: Description provided below the number field.
+* **Key**:Identifier used to map data to the number field (see [Mapping components to Process Variables]({{< ref "#mapping-components-to-process-variables" >}})).
```
[the remark this answers is not in the document]
Comment (MaxTru) [reply content/reference/forms/camunda-forms/_index.md]: Good catch, adjusted.

File: content/reference/forms/camunda-forms/_index.md [hunk 2]
```diff
@@ -27,38 +25,121 @@ Camunda Forms allow you to easily design and configure forms and then embed them
...
+
+## Text
+
+A text component allowing to display static information to the user.
+
+{{< img src="img/form-text.svg" title="Form Text Symbol" >}}
+
+A text component can be configured using the following configuration properties:
+
+* **Text**: Either **Markdown** or **basic HTML** which will be rendered in the Form. Please note that dangerous HTML elements will not be rendered so to prevent the risk of cross-site scripting using Camunda Forms.
```
[the remark this answers is not in the document]
Comment (MaxTru) [reply content/reference/forms/camunda-forms/_index.md:124]: Agreed, added. Thanks for the hint

File: content/reference/forms/camunda-forms/_index.md [hunk 3]
```diff
@@ -27,38 +25,121 @@ Camunda Forms allow you to easily design and configure forms and then embed them
...
 * **Action**: The button can either trigger a **Submit** or a **Reset** action.
   * **Submit**: Submit the form (given there are no validation errors).
   * **Reset**: Reset the form, all user inputs will be lost.
+
+
+# Mapping components to Process Variables
+
+Each Camunda Forms component which allows data manipulation has a **Key** attribute. This attribute is used as an identifier to map data of the respective field (1) during initial loading of the form, and (2) during submission of the form.
+
+When a form is [referenced by a User Task]({{< ref "/user-guide/task-forms/_index.md#camunda-forms" >}}) and viewed in the [Camunda Tasklist]({{< ref "/webapps/tasklist/_index.md" >}}), the key will be used to refer to a [Process Variable]({{< ref "/user-guide/process-engine/variables.md" >}}). This means that the value of the Process Variable will be used to populate the respective component initially and that the value of the component will be saved in the Process Variable during submission of the form.
```
[the remark this answers is not in the document]
Comment (MaxTru) [reply content/reference/forms/camunda-forms/_index.md:186]: True, corrected. Thanks

File: content/reference/forms/camunda-forms/_index.md [hunk 4]
```diff
@@ -27,38 +25,121 @@ Camunda Forms allow you to easily design and configure forms and then embed them
...
 
 A button can be configured using the following configuration properties:
 
 * **Field Label**: Label to be displayed on top of the button.
 * **Action**: The button can either trigger a **Submit** or a **Reset** action.
   * **Submit**: Submit the form (given there are no validation errors).
   * **Reset**: Reset the form, all user inputs will be lost.
+
+
+# Mapping components to Process Variables
```
[the remark this answers is not in the document]
Comment (MaxTru) [reply content/reference/forms/camunda-forms/_index.md:182]: yes - agreed. But let's tackle this later as part of a seperate PR. This PR should aim add adding the new components.

[the remark this answers is not in the document]
Comment (volkergersabeck) [reply content/reference/forms/camunda-forms/_index.md:182]: yes, this is a separate topic

**Review by volkergersabeck — APPROVED** [review]

**Pull request merged**